Makefile

```
@@ -88,6 +91,35 @@ docs: ## Legacy docs target alias
 nixie: ## Validate Mermaid diagrams
 	$(NIXIE) --no-sandbox
 
+integration-test: ## Run testcontainers-based provisioning lifecycle tests
```

checkmake (0.3.2)

[warning] Target body for "integration-test" exceeds allowed length of 5 lines (15).

(maxbodylength)